src/msw/dc.cpp

```diff
@@ -2624,11 +2624,11 @@ static bool AlphaBlt(wxMSWDCImpl* dcDst,
             if ( data )
             {
                 wxAlphaPixelData::Iterator p(data);
-                for ( int y = 0; y < data.GetHeight(); y++ )
+                for ( int yy = 0; yy < data.GetHeight(); yy++ )
```

Comment:
Actually, it's just weird that this is use a for loop instead of using the actual iterator is has since it doesn't even use those values.